editoast/src/views/paced_train.rs

```diff
@@ -57,17 +63,20 @@ enum PacedTrainError {
     #[error("{count} paced train(s) could not be found")]
     #[editoast_error(status = 404)]
     BatchPacedTrainNotFound { count: usize },
+    #[error("Paced train '{paced_train_id}', could not be found")]
+    #[editoast_error(status = 404)]
+    NotFound { paced_train_id: i64 },
```

Comment:
Much better! No need to repeat the "PacedTrain". Maybe the same could be done for BatchPacedTrainNotFound for consistency?